src/components/metadata/MetaButtons.js

```diff
@@ -94,4 +101,5 @@ MetaButtons.propTypes = {
   onConvertClick: PropTypes.func.isRequired,
   onRemoveClick: PropTypes.func.isRequired,
   parentKey: PropTypes.string,
+  isDefaultField: PropTypes.bool.isRequired,
```

Comment:
Why is this a `required` attribute?
IMO, if it was optional, then you needn't have to explicitly `isDefaultField={false}` at so many places.
And if it should be `required`, then please move it above one line so that required parameters are together..

Reply:
good idea, I don't see a reason why it needs to be required. let me update.